lib/galaxy/dependencies/pipfiles/default/pinned-dev-requirements.txt

```diff
@@ -1,5 +1,3 @@
--i https://wheels.galaxyproject.org/simple
```

Comment:
Hrmm, looks like these got dropped when I ran, simply `make update-dependencies` -- @nsoranzo is the expectation that these are manually added to the file, or has something changed since last time we ran this?

Reply:
That doesn't happen for me under Linux, or in Docker.